src/js/content_scripts/input-mapper-utils.js

```diff
@@ -0,0 +1,182 @@
+/*
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+*/
+
+/* eslint-env browser */
+/* global gamepad */
+
+(function (fluid, $) {
+    "use strict";
+
+    fluid.registerNamespace("gamepad.inputMapperUtils");
+
+    /**
+     *
+     * Scroll horizontally across the webpage.
+     *
+     * @param {Object} that - The inputMapper component.
+     * @param {Integer} value - The value of the gamepad input.
```

Comment:
It would be good to describe the units here.  Is it pixels?

You also need to give some thought to how this will work with analogue controls like thumbsticks, as they will keep sending values very rapidly.  So the net effect is that scrolling will be very fast on a thumbstick or other analogue control.  On the flip side, you need some way to ensure that a digital button can be used reasonably without requiring a hundred button presses.

One idea would be to handle axes differently, since those send repeated values by their nature.  You might also make the scale adjustable per button, which would help with triggers as well.

Reply:
Disregard my comments about the analogue controls, I see how you handled that.  You should still describe the units.